docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,130 @@
 Assessment
 ==========
 
-Metrics
--------
+Introduction
+------------
+
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
+
+The steps of the assesment are as follows:
+1. Identify types of harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
+
+We next examine these four steps in more detail.
 
-.. currentmodule:: fairlearn.metrics
+Identify types of harms
+^^^^^^^^^^^^^^^^^^^^^^^
+
+See :ref:`types_of_harms` for a guide to types of fairness-related harms. 
+For example, in a system for screening job applications, qualified candidates 
+that are automatically rejected experience an allocation harm. In a 
+speech-to-text transcription system, disparities in word error rates for 
+different groups may result in harms due to differences in the quality of service.
+Note that one system can lead to multiple harms, and different types of 
+harms are not mutually exclusive. For more information, review 
+Fairlearn's `2021 SciPy tutorial <https://github.com/fairlearn/talks/blob/main/2021_scipy_tutorial/overview.pdf>`_.
+
+Identify the groups that might be harmed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular use case or deployment context. For example, for 
+speech-to-text transcription, this might include groups who speak a regional dialect or people who are a  
+native or a non-native speaker.
+
+It is also important to consider group intersections, for example, in addition
+to considering groups according to gender and groups according to race, it is 
+also important to consider their intersections (e.g., Black women, Latinx 
+nonbinary people, etc.).
+
+Quantify harms
+^^^^^^^^^^^^^^
+
+Define metrics that quantify harms or benefits:
+
+* In job screening scenario, we need to quantify the number of candidates that are classified as "negative" (not recommended for the job), but whose true label is "positive" (they are "qualified"). One possible metric is the false negative rate: fraction of qualified candidates that are screened out. Note that before we attempt to classify candidates, we need to determine the construct validity of the "qualified" status; more information on construct validity can be found in :ref:`construct_validity`.
+
+* For a speech-to-text application, the harm could be measured by disparities in the word error rate for different group, measured by the number of mistakes in a transcript divided by the overall number of words.
+
+Note that in some cases, harms might not be readily quantifiable. 
+In these cases, we might choose to use a closely related variable,
```

Comment:
It was not immediately clear to me that the quantifiability of the harm is directly related to a missing variable. An additional sentence to explain this might be useful (perhaps we can also relate it to some of the abstraction traps?).

Reply:
Ah, I understand. I've edited the sentences as follows. Updated text is italicized.

"Note that in some cases, harms might not be readily quantifiable. 
_Occasionally, another variable in our dataset provides a close
approximation to the phenomenon we seek to measure._
In these cases, we might choose to use that closely related variable,
often called a "proxy", to stand in for the missing variable. 
For example, suppose that in 
the job screening scenario, we have data on whether the candidate passes 
the first two stages, but not if they are ultimately recommended for the 
job. 

As an alternative to the unobservable final recommendation, we could
therefore measure the harm using the proxy variable indicating whether
the candidate passes the first stage of the screen.
If you choose to use a proxy variable to 
represent the harm, check the proxy variable regularly to ensure it 
remains useful over time. Our section on :ref:`construct_validity`
describes how to determine whether a  
proxy variable measures the intended construct in a meaningful 
and useful way. _Also be careful to ensure that the proxy measure 
you choose does not abstract away the social context  
of the problem you seek to solve. 
For more on abstraction traps, review :ref:`abstraction_traps`._"

Please let me know if this lines up with what you're thinking!